src/_gettsim_tests/test_data/erwerbsm_rente/2001/hh_id_1.yaml

```diff
@@ -0,0 +1,128 @@
+---
+info:
+  note: ''
+  source: Own calculations
+inputs:
+  provided:
+    p_id:
+      - 1
+      - 2
+      - 3
+      - 4
+      - 5
+    geburtsjahr:
+      - 1940
+      - 1941
+      - 1970
+      - 1980
+      - 1980
+    geburtsmonat:
+      - 2
+      - 4
+      - 6
+      - 8
+      - 10
+    jahr_renteneintr:
+      - 2001
+      - 2001
+      - 2001
+      - 2001
+      - 2001
+    monat_renteneintr:
+      - 10
+      - 8
+      - 6
+      - 4
+      - 2
+    erwerbsgemindert:
+      - true
+      - true
+      - true
+      - true
+      - true
+    entgeltp_west:
+      - 15.0
+      - 18.0
+      - 14.0
+      - 3.0
+      - 4.0
+    entgeltp_ost:
+      - 0.0
+      - 0.0
+      - 0.0
+      - 0.0
+      - 0.0
+    wohnort_ost:
+      - false
+      - false
+      - false
+      - true
+      - true
+    teilw_erwerbsm_rente:
+      - false
+      - false
+      - false
+      - false
+      - true
```

Comment:
This seems weird. From the docstrings I inferred that `erwerbsgemindert` and `teilw_erwerbsgemindert` (new suggestion) are mutually exclusive?